static/app/components/forms/model.tsx

```diff
@@ -13,7 +13,14 @@ import {defined} from 'sentry/utils';
 type Snapshot = Map<string, FieldValue>;
 type SaveSnapshot = (() => number) | null;
 
-export type FieldValue = string | number | boolean | Choice | Set<string> | undefined; // is undefined valid here?
+export type FieldValue =
+  | string
+  | string[]
```

Comment:
`string[]` is the new one here, used here https://github.com/getsentry/sentry/blob/4b54b4e286cec03905b6aea049dd1e60dd2659cf/static/app/views/settings/organizationDeveloperSettings/resourceSubscriptions.tsx#L31 which declaring the context makes this more strongly typed